clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -183,6 +222,13 @@ public ListTopicsResult listTopics() {
      * This is a convenience method for #{@link AdminClient#describeTopics(Collection, DescribeTopicsOptions)} with
      * default options. See the overload for more details.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link DescribeTopicsResult#values() values()} method of the  returned {@code DescribeTopicsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidTopicException}
+     *     if the topic name can't be represented in the request, or if it is not found</li>
```

Comment:
This is not correct.  We return `UnknownTopicOrPartitionException` if the topic is not found.